.env

```diff
@@ -0,0 +1,7 @@
+MONGOHQ_URL=localhost
+FB_CLIENT_ID=
+FB_CLIENT_SECRET=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
```

Comment:
Env keys without values are not ideal. It's either they have a dummy value or they aren't there at all